lms/views/api/exceptions.py

```diff
@@ -128,7 +128,9 @@ def external_request_error(self):
                 "body": self.context.response_body,
             },
         )
-        sentry_sdk.set_context("validation_errors", self.context.validation_errors)
+
+        validation_errors = getattr(self.context, "validation_errors", None)
```

Comment:
Need some conditions here as now 
ExternalRequestError and ExternalAsyncRequestError differ. No message or validation errors in the latter.

Here went for some conditions in the same method instead of two different ones. Could always split it the diverge more.